packages/atomic/src/components/atomic-result/atomic-result.pcss

```diff
@@ -1,5 +1,236 @@
-@import "tailwindcss/base";
+@import 'tailwindcss/base';
+@import '../../global/global.pcss';
```

Comment:
Without adding this, the lack of `atomic-result-quickview` in a template would cause tailwind classes to be missing.

Reply:
base was there to "reset" the CSS of the content.

Content of global.pcss:
```
@import 'tailwindcss/base';
@import 'tailwindcss/utilities';

:host {
  display: block;
  @apply font-sans font-normal text-base;
}
.... more
```

Note that all of Tailwind is added in dev, in prod we strip whatever is not used. When you add utilities, you'll add a few classes, maybe `m-1`, `text-primary` etc etc. In the end, an incomplete set of random classes would be available to our customers, causing confusion.

I would: 
- not add any tailwind utilities here in atomic-result
- not add any tailwind utilities in result template components that don't have shadow DOM

Perhaps we can investigate:
- adding shadow DOM and tailwind to all the result template components to style their internals
- add css parts and css variables to these component for customization
- if we had parts to these components, we won't be able to access them from the root html, we maybe could
      1.  look into this experimental feature to expose parts https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes#attr-exportparts (bad idea IMO)
      2. if customer have a `<style>` tag inside their template and then inside the atomic-result's shadow DOM, the parts should work (better idea IMO)

Reply:
If we add shadow DOM to result templates then I believe we should also replace all the new section `class` with an `id`.

Reply:
When I wrote result template components i mean these, not atomic-result-template which purpose is to define templates:

![Screen Shot 2021-06-18 at 3 12 17 PM](https://user-images.githubusercontent.com/4923043/122606833-93736200-d047-11eb-8fa5-12302b3f9c1b.png)

Section should have class, there's no drawbacks